buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -83,9 +83,9 @@ object Deps {
 
         object Firebase {
```

Comment:
Unfortunately, we don't use these varaibles for now.
please check `root/dependencies.gradle`